src/components/social-share.js

```diff
@@ -33,7 +33,7 @@ class SocialShareBase extends React.Component {
 
     return React.createElement(this.props.template, Object.assign({
       fbUrl: `https://www.facebook.com/sharer.php?u=${encodeURIComponent(fullUrl)}`,
-      twitterUrl: `https://twitter.com/intent/tweet?url=${encodeURIComponent(fullUrl)}&text=${encodeURIComponent(this.props.title)}&hashtags=${hashtags}`,
+      twitterUrl: `https://twitter.com/intent/tweet?url=${encodeURI(fullUrl)}&text=${encodeURIComponent(this.props.title)}&hashtags=${hashtags}`,
```

Comment:
We can pass the encoded Url from the app level? 
```
<SocialShare
            template={ShareAllTemplate}
            title={storyData.headline}
            fullUrl={encodeURI(story.url)}
          />
```

Reply:
because it is easier to fix in the implementation level rather than modifying ever use case in the app level

Reply:
Currently, Prothomalo and Prabhatkbhar, They already passing it in encoded form. @sivteck

Reply:
awesome